cmd/deploy.go

```diff
@@ -21,29 +21,52 @@ var deployCmd = &cobra.Command{ //nolint:gochecknoglobals
 	Args:  cobra.ExactArgs(1),
 	Run: func(cmd *cobra.Command, args []string) {
 		projectId := flags.GetProjectId()
+		if projectId == "" {
+			logger.Fatal("Must provide a project ID in the --project flag")
+
+			return
```

Comment:
both `logger.Fatal` and `logger.FatalErr` exit the process, so there's no need for these return statements.

Reply:
`os.Exit` did not allow for the deferred file removal to work, so I removed it and chose to return instead. But I've added the `os.Exit` back in now & changed all the logger signatures to as they were because we don't need to remove any files.

Reply:
Nice, thanks for that tip. Will come in handy!